packages/test/amplify_test/lib/amplify_test.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+library amplify_test;
```

Comment:
For the different main test files, can we add some brief comments on what is being exported for each?  While it's clear now, I can imagine in the future where it might not be readily remembered what each test package is supposed to export.

Reply:
Updated 👍